File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
    
    if(a == b): return a

    first = a
    last = b
    if(b < a):
        first = b
        last = a
        
    for i in range(first, last+1):
        answer += i
        
    return answer

File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;

int* swap(int a, int b){
    
    int* array = new int[2];
    int tmp = a;
    array[0] = b;
    array[1] = tmp;
    
    return array;
}

long long solution(int a, int b) {
    long long answer = 0;
    
    if(a == b){
        return a;
    }else{
        if(a > b){
            int* t = swap(a, b);
            a = t[0];
            b = t[1];
        }
        for(int i=a; i<=b; i++){
            answer += i;
        }
    }
    return answer;
}